paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/EditCardDetailsInteractor.kt

```diff
@@ -0,0 +1,95 @@
+package com.stripe.android.paymentsheet.ui
+
+import androidx.compose.runtime.Immutable
+import com.stripe.android.CardBrandFilter
+import com.stripe.android.model.CardBrand
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.paymentsheet.CardUpdateParams
+import kotlinx.coroutines.flow.StateFlow
+
+internal typealias CardDetailsCallback = (CardUpdateParams?) -> Unit
```

Comment:
Calling this "CardDetailsCallback"  introduces "Details" as new nomenclature. It seems like we avoid this by just changing the name of this to: `CardUpdateParamsCallback`